jenkins/ci.suse.de/cloud-mkphyscloud-qa-scenario-2a.yaml

```diff
@@ -270,8 +270,10 @@
 
               chmod +x scripts/iscsictl.py
 
+              ./scripts/iscsictl.py --service target --host \$(hostname) --no-key
+
               for node in {controller1,controller2,data1,data2,network1,network2} ; do
-                ./scripts/iscsictl.py --service initiator --target_host $sbd_ip --host \$node --no-key --id id-qa$hw_number
+                ./scripts/iscsictl.py --service initiator --target_host \$sbd_ip --host \$node --no-key
```

Comment:
that added `\` means, `$sbd_ip` is passed into iscsictl.py as a string - I would be surprised if it could handle that.